dbcs/honda_crv_touring_2016_can.dbc

```diff
@@ -0,0 +1,312 @@
+VERSION ""
```

Comment:
I've been meaning add some docs to the comma wiki about it. Will try to get to it.